src/controller/dac_chain.h

```diff
@@ -0,0 +1,28 @@
+#pragma once
+
+// TODO: Implement a mechanism to handle and secure DAC/PAI/PAA certificates and corresponding keypairs
+// Currently this is hard-coded to demonstrate Attestation Flow
```

Comment:
Hardcoding is OK if there are additional abstractions that allow smooth transition to no longer use hard-coding. This hardcoded value is accessed directly, and knowledge of its size is necessary, as well. If this header is included from more than one compilation unit, there will be a clash at link time for a duplicate symbol definition. Please provide some accessor APIs and hide this constant in a .cpp file.

Reply:
removed hard-coded buffers